plugins/inputs/win_services/win_services_test.go

```diff
@@ -225,3 +226,25 @@ func TestGatherContainsTag(t *testing.T) {
 		acc1.AssertContainsTaggedFields(t, "win_services", fields, tags)
 	}
 }
+
+func TestExcludingNamesTag(t *testing.T) {
+	winServices := &WinServices{
+		Log:                  testutil.Logger{},
+		ServiceNames:         []string{"*"}, // '*' == default, all included
```

Comment:
Can we please avoid setting this explicitly in the test to check for backward compatibility?

Reply:
removed as well. thanks!